frontend/src/employee-frontend/components/reports/AssistanceNeedDecisionsReport.tsx

```diff
@@ -0,0 +1,216 @@
+import Title from 'lib-components/atoms/Title'
+import ReturnButton from 'lib-components/atoms/buttons/ReturnButton'
+import Combobox from 'lib-components/atoms/dropdowns/Combobox'
+import { Container, ContentArea } from 'lib-components/layout/Container'
+import {
+  SortableTh,
+  Table,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr
+} from 'lib-components/layout/Table'
+
+import { getAssistanceNeedDecisionsReport } from '../../api/reports'
+import { useTranslation } from '../../state/i18n'
+import { distinct } from '../../utils'
+import { AssistanceNeedDecisionStatusChip } from '../child-information/assistance-need/decision/common'
+
+import { FilterLabel, FilterRow } from './common'
+
+const Wrapper = styled.div`
+  width: 100%;
+`
+
+const BorderedTbody = styled(Tbody)`
+  border: 1px solid ${(p) => p.theme.colors.grayscale.g15};
+`
+
+const Highlight = styled.div`
+  background-color: ${(p) => p.theme.colors.status.success};
+  width: 6px;
+  position: absolute;
+  left: 0.5px;
+  top: 0.5px;
+  bottom: 0.5px;
+`
+
+const RelativeTr = styled(Tr)`
+  position: relative;
+`
+
+export default React.memo(function AssistanceNeedDecisionsReport() {
+  const { i18n } = useTranslation()
+  const [report, setReport] = useState<
+    Result<AssistanceNeedDecisionsReportRow[]>
+  >(Loading.of())
+  const [careAreaFilter, setCareAreaFilter] = useState<string>()
+
+  useEffect(() => {
+    setReport(Loading.of())
+    void getAssistanceNeedDecisionsReport().then(setReport)
+  }, [])
+
+  const [sortColumn, setSortColumn] =
+    useState<keyof AssistanceNeedDecisionsReportRow>('sentForDecision')
+  const [sortDirection, setSortDirection] = useState<SortDirection>('DESC')
+
+  const sortBy = useCallback(
+    (column: keyof AssistanceNeedDecisionsReportRow) => {
+      if (sortColumn === column) {
+        setSortDirection(sortDirection === 'ASC' ? 'DESC' : 'ASC')
+      } else {
+        setSortColumn(column)
+        setSortDirection(column === 'sentForDecision' ? 'DESC' : 'ASC')
+      }
+    },
+    [sortColumn, sortDirection]
+  )
+
+  const decisionRows = useMemo(
+    () =>
+      report
+        .map((rs) =>
+          orderBy(
+            rs.filter(
+              (row) => !careAreaFilter || row.careAreaName === careAreaFilter
+            ),
+            [sortColumn],
+            [sortDirection === 'ASC' ? 'asc' : 'desc']
+          )
+        )
+        .getOrElse([]),
+    [report, careAreaFilter, sortColumn, sortDirection]
+  )
+
+  const navigate = useNavigate()
+
+  return (
+    <Container>
+      <ReturnButton label={i18n.common.goBack} />
+      <ContentArea opaque>
+        <Title size={1}>{i18n.reports.assistanceNeedDecisions.title}</Title>
+        <FilterRow>
+          <FilterLabel>{i18n.reports.common.careAreaName}</FilterLabel>
+          <Wrapper>
+            <Combobox
+              items={[
+                { value: '', label: i18n.common.all },
+                ...report
+                  .map((rs) =>
```

Comment:
This seems like a minor thing, but I think it's important to avoid very short variable/parameter names. The reason is that the reader of the code is forced to stop, go back a bit, inspect the type of `report` and `rs` in this case and then map that knowledge onto what the result of this code will be.

As code is written once and read like a million times, it makes sense to optimize for readability.